components/controls/ClipboardButton.tsx

```diff
@@ -67,23 +66,23 @@ export default class ClipboardButton extends React.PureComponent<Props, State> {
 
   showTooltip = () => {
     if (this.mounted) {
-      this.setState({ tooltipShown: true });
+      this.setState({ isCopied: true });
       setTimeout(() => {
         if (this.mounted) {
-          this.setState({ tooltipShown: false });
+          this.setState({ isCopied: false });
```

Comment:
I'm not sure it's really working as expected... So now the current workflow is:
1. mouse over we display: "Click to copy to clipboard"
2. when I click we display: "Copied to clipboard"
3. after 1 second the message change back to: "Click to copy to clipboard"

I'm really not sur about that last part, I think this timer was used to display and hide after a little time, now it's used to change the text... Should we really change the text based on a timer ? 

Also I think we should keep the flag to force show the tooltip for a second (better ux when using a keyboard, the tooltip will be displayed).

Reply:
Mixing displaying on hover and with a timer is going to introduce a new layer of complexity. I personally don't mind having the tooltip remaining visible. The user will probably move away her mouse anyway.